internal/gengapic/client_init.go

```diff
@@ -29,7 +29,7 @@ import (
 func (g *generator) clientHook(servName string) {
 	p := g.printf
 
-	p("var new%sClientHook clientHook", servName)
+	p("var new%sGRPCClientHook clientHook", servName)
```

Comment:
This will actually break Spanner, who define a hook for the clients today. Can we leave this one named the same and maybe put a comment on it to the affect of `// newClientHook is a client initialization hook for gRPC client implementations`.

When Rest comes along, we can generate a separate hook like `newRestClientHook`. WDYT?

Reply:
I'm in favor